File: src/main/java/com/appointmed/dto/DoctorProfileUpdateRequest.java
package com.appointmed.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class DoctorProfileUpdateRequest {
    private String specialization;
    private String licenseNumber;
    private String phone;
    private String clinicAddress;
    private BigDecimal consultationFee;
}

File: mobile/app/src/main/java/com/appointmed/mobile/data/model/DoctorProfileUpdateRequest.kt
package com.appointmed.mobile.data.model

data class DoctorProfileUpdateRequest(
    val specialization: String? = null,
    val licenseNumber: String? = null,
    val phone: String? = null,
    val clinicAddress: String? = null,
    val consultationFee: Double? = null
)
